unstructured/ingest/pipeline/encrypt/fernet.py

```diff
@@ -0,0 +1,20 @@
+from dataclasses import dataclass
+
+from cryptography.fernet import Fernet
+
+from unstructured.ingest.pipeline.encrypt.interfaces import Encryption
+
+
+@dataclass
+class FernetEncryption(Encryption):
+    key: str
+    encoding: str = "UTF-8"
```

Comment:
Is this supposed to be used in the `decode()` below?
Also can you add a comment on why we encode then decode. I'm assuming self.fernet.encrypt encodes into something that is not UTF-8? Like bytes or something?